src/structures/Sticker.js

```diff
@@ -0,0 +1,88 @@
+'use strict';
+
+const Base = require('./Base');
+const { StickerFormatTypes } = require('../util/Constants');
+const Snowflake = require('../util/Snowflake');
+
+/**
+ * Represents a Sticker.
+ * @extends {Base}
+ */
+class Sticker extends Base {
+  constructor(client, sticker) {
+    super(client);
+    /**
+     * The ID of the sticker
+     * @type {Snowflake}
+     */
+    this.id = sticker.id;
+
+    /**
+     * The ID of the sticker's image
+     * @type {string}
+     */
+    this.asset = sticker.asset;
+
+    /**
+     * The description of the sticker
+     * @type {string}
+     */
+    this.description = sticker.description;
+
+    /**
+     * The format of the sticker
+     * @type {StickerFormatTypes}
+     */
+    this.format = StickerFormatTypes[sticker.format_type];
+
+    /**
+     * The name of the sticker
+     * @type {string}
+     */
+    this.name = sticker.name;
+
+    /**
+     * The ID of the pack the sticker is from
+     * @type {Snowflake}
+     */
+    this.packID = sticker.pack_id;
+
+    /**
+     * The ID of the sticker preview image
+     * @type {?string}
+     */
+    this.previewAsset = sticker.preview_asset;
+
+    /**
+     * An array of tags for the sticker, if any
+     * @type {?string[]}
+     */
+    this.tags = sticker.tags ? sticker.tags.split(', ') : null;
+
+    /**
+     * A link to the sticker.
```

Comment:
```suggestion
     * A link to the sticker
```

Reply:
looks like Kyra said to add periods?

Reply:
the general style is method descriptions end with them, but property descriptions do not